arabic/unpacked/html.js

```diff
@@ -0,0 +1,90 @@
+MathJax.Hub.Config({
+  'HTML-CSS': {
+    undefinedFamily: 'Amiri',
+    styles: {
+      '.mfliph': {
```

Comment:
You might want to make this '.MathJax .mfliph' just in case the pages uses class `mfliph` itself (we have run into clashes before).  Similarly for `.mar` below.